src/main/java/baritone/pathing/movement/CalculationContext.java

```diff
@@ -85,6 +87,10 @@ public CalculationContext() {
         this.worldBorder = new BetterWorldBorder(world.getWorldBorder());
     }
 
+    public final IBaritone getBaritone() {
+        return this.baritone;
```

Comment:
REEEEEEEEEEEee nothing else in this class uses this.

when i come into your house (the `cache` package), i follow your rules, i take off my shoes, i use `this.`

when you come into my house, i expect you to follow my rules (keep your shoes on at all times) and don't use `this.`